### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Veteran Status, Veteran Status MVP Page

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#74327]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- All of this looks really good! I think this is going in a great direction.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Coding:** When this is coded, the image with the sample status card needs to have alt text, but since it's just dummy information it doesn't have to be super high-fidelity. My suggestion would be something like: `Sample Veteran status card name, date of birth, disability rating, and dates of service.` For specific wording I'll always defer to the content folks though. 😄 
- **Must:** For the download link, if it opens in a new tab be sure to include some kind of text indicating that it opens in a new tab to warn users. Visible text is best because it's accessible to everyone, but it's also fine to just include it as screen reader only text, eg. `<span class="sr-only">(opens in a new tab)</span>`.
- **Should:** I spent a lot of time thinking about this download link versus download button issue, and I think here's where I'm landing: It _should_ be a button rather than a link, but this wouldn't be something I would log at staging review.
   
   The blob URL solution to make this a link is a great idea, and I think the arguments you make about opening it in a new tab and allowing users to choose where in their file system to download to make sense. But from a material honesty perspective, I think the button is the right way to go. The card isn't a persistent resource with a nice URL, it's being generated via script for this user when the user clicks the element. That's more action-y than navigation-y, which falls under the button umbrella. (Ideally the text of the button would reflect that too, eg. "Generate and download your Veteran status card (PDF)" --- but that's a VERY long button for our button styles.)
   
   But... it's enough of an edge case that I don't think it's ultimately a big deal. If you make it a button I'll be slightly happier when I see you at staging, but I'm not going to log it as an issue either way.
